src/context/WalletProvider/NativeWallet/components/NativeImport.tsx

```diff
@@ -16,6 +16,7 @@ import { Text } from 'components/Text'
 export const NativeImport = ({ history }: RouteComponentProps) => {
   const onSubmit = async (values: FieldValues) => {
     try {
+      values.mnemonic = values.mnemonic.toLowerCase().trim()
       const vault = await Vault.create()
       vault.meta.set('createdAt', Date.now())
       vault.set('#mnemonic', values.mnemonic)
```

Comment:
Just a gut feeling, but I'm paranoid about storing that result.
```suggestion
      const vault = await Vault.create()
      vault.meta.set('createdAt', Date.now())
      vault.set('#mnemonic', values.mnemonic.toLowerCase().trim())
```

Reply:
I prefer your approach too, @mrnerdhair 👍